### PR TITLE
Update the description of the zfs cheat sheet

### PR DESCRIPTION
**Pull request by s7nio**
###### What does your Pull Request do (check all that apply)?

Choose the most relevant items and use the following title template to name
your Pull Request.

- [ ] New Instant Answer
    - [ ] Cheat Sheets: **`New {Cheat Sheet Name} Cheat Sheet`**
    - [ ] Other: **`New {IA Name} Instant Answer`**
- [x] Improvement
    - [ ] Bug fix: **`{IA Name}: Fix {Issue number or one-line description}`**
    - [x] Enhancement: **`zfs_cheat_sheet: update the description of the zfs cheat sheet`**
- [ ] Non–Instant Answer
    - [ ] Other (Role, Template, Test, Documentation, etc.): **`{GoodieRole/Templates/Tests/Docs}: {Short Description}`**

###### Description of changes

Update the description of the zfs cheat sheet

###### Which issues (if any) does this fix?

Fixes #NNNN - how specifically does it fix the issue?

###### People to notify (interested parties)

---

Instant Answer Page: https://duck.co/ia/view/zfs_cheat_sheet

Maintainer: @s7nio

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/zfs.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "id": "zfs_cheat_sheet",
     "name": "ZFS",
-    "description": "The combined file system and logical volume manager.",
+    "description": "The combined file system and logical volume manager. Data are important - be careful with it.",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/zfs.json]: Uhh... I'm not sure this really adds to the cheat sheet. I don't think we really need to warn the user.

@mintsoft Any thoughts?

Comment (mintsoft) [inline share/goodie/cheat_sheets/json/zfs.json]: @GuiltyDolphin Meh; not fussed!

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=ZFS Cheat Sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/zfs_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/zfs.json (hunk 1 above)
Comment (s7nio) [inline share/goodie/cheat_sheets/json/zfs.json]: I agree with @GuiltyDolphin - strict of the cheat sheet perspective. But you can damage/loss the data with some commands, which is not cool.

I've not read something about it in the docs. Any further opinions?

Comment (mintsoft) [inline share/goodie/cheat_sheets/json/zfs.json]: @s7nio I've already merged this, I don't forsee any real problems. From my POV, it's "your" cheatsheet, so your call :)

**Pull request merged**